test/into.js

```diff
@@ -0,0 +1,37 @@
+var assert = require('assert');
+
+var R = require('..');
+
+
+describe('into', function() {
+    var add = R.curry(function(a, b) {return a + b;});
+    var addXf = {
+        step: add,
+        init: R.always(0),
+        result: R.I
+    };
+
+    it('is automatically curried', function() {
+        var intoArray = R.into([]);
+        var add2 = R.map(add(2));
+        var result = intoArray(add2);
+        assert.deepEqual(result([1, 2, 3, 4]), [3, 4, 5, 6]);
+    });
+
+    it('allows custom transformer', function() {
+        var intoSum = R.into(addXf);
+        var add2 = R.map(add(2));
+        var result = intoSum(add2);
+        assert.deepEqual(result([1, 2, 3, 4]), 18);
+    });
+
+    it('correctly reports the arity of curried versions', function() {
+        var sum = R.into([], R.map(add));
+        assert.strictEqual(sum.length, 1);
+    });
+
+    it('throws on zero arguments', function() {
+        assert.throws(R.into, TypeError);
+        assert.throws(R.into([]), TypeError);
+    });
```

Comment:
none of these tests the primary use case, which is pouring one collection into another. the closest is 'allows custom transformer', but that is a special case. Can you add tests for use without custom transformers, i.e. using what the lib provides? Otherwise this looks great